src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -623,6 +655,7 @@ internal enum LoadPhase
     {
         None,
         PreBuild,
-        PostBuild
+        PostBuild,
+        PostPostBuild,
```

Comment:
i want to mean the model loaded after "PostBuild" step

---
In reply to: [77627790](https://github.com/dotnet/docfx/pull/670#discussion_r77627790) [](ancestors = 77627790,77580084,77579915)